bitnami/spring-cloud-dataflow/Chart.lock

```diff
@@ -4,12 +4,12 @@ dependencies:
   version: 1.4.1
 - name: mariadb
   repository: https://charts.bitnami.com/bitnami
-  version: 9.3.4
+  version: 9.3.5
 - name: rabbitmq
   repository: https://charts.bitnami.com/bitnami
-  version: 8.11.2
+  version: 8.11.3
 - name: kafka
   repository: https://charts.bitnami.com/bitnami
-  version: 12.9.5
-digest: sha256:34f2312eb4fb40f47fd140771efa52914a739f9049ec214f20bfcd9d19547608
-generated: "2021-03-04T02:22:15.749764419Z"
+  version: 12.11.1
+digest: sha256:4704eb5697ccacaf418385efc937b0f5cacafc59cb3966c9883237ec11c010c4
+generated: "2021-03-17T00:31:31.6493933-04:00"
```

Comment:
Noticed this file is updated because dependencies are not fixed to a specific version. Should I keep this file?

Reply:
It is fine to update it, we use the latest version for the major don't worry about it